api/src/main/scala/hmda/api/http/institutions/SubmissionPaths.scala

```diff
@@ -141,4 +141,39 @@ trait SubmissionPaths
       }
 
     }
+
+  def submissionSingleEditPath(institutionId: String) =
+    path("filings" / Segment / "submissions" / IntNumber / "edits" / Segment) { (period, submissionId, editType) =>
+      val path = s"institutions/$institutionId/filings/$period/submissions/$submissionId/edits/$editType"
+      extractExecutionContext { executor =>
+        implicit val ec: ExecutionContext = executor
```

Comment:
I tried it locally to make sure I wasn't lying, and I had luck making a method with this signature:
```scala
  def validationState(subId: SubmissionId)(implicit ec: ExecutionContext): Future[HmdaFileValidationState] = {
```